Products/ZenModel/ZenossInfo.py

```diff
@@ -279,6 +281,7 @@ def getZopeVersion(self):
         major, minor, micro, status, release = version.getZopeVersion()
         return Version(name, major, minor, micro)
 
+    @FunctionCache('getZenossRevision')
```

Comment:
5.x doesn't run on SVN, this cmd will never work.  The method should either be removed or always return an empty string